pkgs/development/libraries/fontconfig/default.nix

```
@@ -24,11 +26,12 @@ stdenv.mkDerivation (finalAttrs: {
     "out"
   ]; # $out contains all the config
 
-  src = fetchurl {
-    url =
-      with finalAttrs;
-      "https://www.freedesktop.org/software/fontconfig/release/${pname}-${version}.tar.xz";
-    hash = "sha256-ajPcVVzJuosQyvdpWHjvE07rNtCvNmBB9jmx2ptu0iA=";
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "fontconfig";
+    repo = "fontconfig";
+    tag = finalAttrs.version;
+    hash = "sha256-R2y5H4JbtpCLRW7BvLXj5H6NrzVZf6+DsaQCuIhibzM=";
```

We do not really have the ancient `docbook2man` packaged. I am attempting to modernize the toolchain in https://gitlab.freedesktop.org/fontconfig/fontconfig/-/merge_requests/304 but until then, you should use the dist tarball, not the one generated by GitLab. The latter lacks the pre-generated man pages.

```suggestion
  src = fetchurl {
    url = "https://gitlab.freedesktop.org/api/v4/projects/890/packages/generic/fontconfig/${finalAttrs.version}/fontconfig-${finalAttrs.version}.tar.xz";
    hash = "";
```

---

Thanks for sharing it! The URL looks working perfectly. :pray: 
So changed to tarball again in https://github.com/NixOS/nixpkgs/pull/401037/commits/b3bb63f4410df02280b31e5e1ca711c45bbfe73f.

And added a man pages checks in https://github.com/NixOS/nixpkgs/pull/401037/commits/f80a788128b2f8235b62f08cc1b4eed3077b308a.